Yoast/javascript
# Implement new classes to english

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Implements the new Clause and SentenceNew class to English folder, adds English Clause class, and adjusts related specs.

## Relevant technical choices:

* `getPassiveVoiceResult`, `getClauses` and `getClausessplitOnStopwords` (and their spec files) are created to accommodate the new changes/structure in Clause and SentenceNew class.
* The `setClausePassiveness` method is deleted from src/values/Clause class → the functionality from this check is moved to English Clause class (checkParticiples())
* `getParticiples` helper is adjusted to return an array of participles (string) instead of an array of participle objects and adjust the specs
* All the participle checks are moved from EnglishParticiple to EnglishClause and EnglishParticiple file and the specs are deleted
* `IsSentencePartPassive` helper and the specs are deleted, also delete it from the Researcher.
* The empty  branch in premium-configuration with the same name should be deleted once this PR is merged

## Test instructions

This PR can be tested by following these steps:

*  run `yarn test` and make sure that everything pass (except `runFullTextTests` for English)

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LINGO-753
